File: app/src/main/java/com/example/foxizz/navigation/util/SearchItem.java
package com.example.foxizz.navigation.util;

public class SearchItem {
    private String targetName;
    private String address;
    private Double distance;

    public SearchItem(String targetName, String address, Double distance) {
        super();
        this.targetName = targetName;
        this.address = address;
        this.distance = distance;
    }
    public SearchItem() {
        super();
    }

    public String getTargetName() {
        return targetName;
    }
    public void setTargetName(String targetName) {
        this.targetName = targetName;
    }

    public String getAddress() {
        return address;
    }
    public void setAddress(String address) {
        this.address = address;
    }

    public Double getDistance() {
        return distance;
    }
    public void setDistance(Double distance) {
        this.distance = distance;
    }

}

File: app/src/main/java/com/navigation/foxizz/data/SearchItem.kt
package com.navigation.foxizz.data

import com.baidu.mapapi.model.LatLng

/**
 * 搜索目标信息类
 */
data class SearchItem(
        var uid: String = "", //唯一地址标识
        var latLng: LatLng? = null, //坐标
        var targetName: String = "", //目标名
        var address: String = "", //目标地址
        var distance: Double = 0.0 //与目标的距离
)